tests/client/disco/components/TestAddon.js

```diff
@@ -48,15 +48,9 @@ function renderAddon({ setCurrentStatus = sinon.stub(), ...props }) {
 
 describe('<Addon />', () => {
   describe('<Addon type="extension"/>', () => {
-    let root;
-
-    beforeEach(() => {
-      root = renderAddon(result);
-    });
-
     it('renders a default error overlay with no close link', () => {
       const data = { ...result, status: ERROR, setCurrentStatus: sinon.stub() };
-      root = renderAddon(data);
+      const root = renderAddon({ addon: data, ...data });
```

Comment:
There's an issue filed for the opposite. The data from `addon` is used but it's being spread into the rest of the props as well. I think it would be better to have the addon props in `addon` instead.